internal/otelarrow/netstats/netstats_test.go

```diff
@@ -13,13 +13,88 @@ import (
 	"go.opentelemetry.io/collector/exporter"
 	"go.opentelemetry.io/collector/receiver"
 	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/instrumentation"
 	"go.opentelemetry.io/otel/sdk/metric"
 	"go.opentelemetry.io/otel/sdk/metric/metricdata"
 	"go.opentelemetry.io/otel/sdk/resource"
 	sdktrace "go.opentelemetry.io/otel/sdk/trace"
 	"google.golang.org/grpc/stats"
 )
 
+func dropView(instrument metric.Instrument) metric.View {
+	return metric.NewView(
+		instrument,
+		metric.Stream{
+			Aggregation: metric.AggregationDrop{},
+		},
+	)
+}
+
+// TODO: This will be exposed by each component. Remove this function and use
+// the publicly exposed view once this is possible to do.
```

Comment:
Is there an existing issue we can tie this to as a reminder?

Reply:
Right, that would be open-telemetry/opentelemetry-collector/issues/11754, added